spec/score_card_spec.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require 'score_card'
+
+describe ScoreCard do
+  let(:bowl) { ScoreCard.new }
+  let(:frame) { double :frame }
+  describe '#frames' do
+    it 'should be 1 by default' do
+      expect(bowl.frames).to eq []
+    end
+  end
```

Comment:
Perhaps also include a test to check that frames are added to it after the end of a frame? Or is this implicitly tested by the tests for `#roll` below?